cql3/statements/select_statement.cc

```diff
@@ -60,6 +60,8 @@ namespace cql3 {
 
```

Comment:
Shouldn't this commit come after all the others, which gradually add support for full native reverse support?

Reply:
Depends. I decided to apply changes from top to bottom. Starting from here and propagating the support further down through storage_proxy, database, table up to the readers. Then this patch comes first.